.trivyignore

```diff
@@ -0,0 +1,2 @@
+CVE-2023-39325
```

Comment:
pls patch with https://github.com/kubernetes-sigs/blob-csi-driver/pull/1098, so we don't need this file any more.

Reply:
done. combined #1098 in this pr.